src/supermarket/app/workers/fieri_notify_worker.rb

```diff
@@ -21,9 +21,23 @@ def perform(cookbook_version_id)
     data = {
       'cookbook_name' => cookbook_version.name,
       'cookbook_version' => cookbook_version.version,
-      'cookbook_artifact_url' => cookbook_version.tarball.url
+      'cookbook_artifact_url' => cookbook_artifact_url(cookbook_version)
     }
 
     response = Net::HTTP.post_form(uri, data)
   end
+
+ def cookbook_artifact_url(cookbook_version)
+    if s3_configured?
+      cookbook_version.tarball.url
+    else
+      "#{Supermarket::Host.full_url}#{cookbook_version.tarball.url}"
+    end
+  end
```

Comment:
Going to push to the CookbookVersion model, as urls are unique for each version.